### ISSUE TITLE
Special character in article tag redirects to the wrong page

### ISSUE THREAD

**Issue by Cwpute** [HermanMartinus/bearblog#94]
In one of my published articles, i used a tag with a special character: `#5senset+`
But on the article page, when i click said tag, it redirects me to a filtered list of articles containing the tag `#5senset` without the `+`, and said list doesn't include the article which i tagged with `#5senset+`.

I don't know whether it's an error on my side because i used a `+` in the tag, or an error in the automatically-generated page for the filtered list of articles which didn't include the `+` where it should have. Either way, it is probably not the intended behavior ! :)

**Comment by HermanMartinus** [comment]
Yeah, in this case the tag manager doesn't allow special characters. I'll make a note to put a warning there or something. Thanks for pointing it out
